tests/test_data/core/core__encounter.txt

```diff
@@ -1,50 +1,50 @@
-({'id': None, 'code': 'AMB', 'display': None, 'system': 'http://terminology.hl7.org/CodeSystem/v3-ActCode', 'userSelected': None, 'version': None}, 'AMB', 'ambulatory', 'finished', None, None, None, None, None, None, None, None, 15, datetime.date(2018, 6, 3), datetime.date(2018, 6, 3), datetime.date(2018, 5, 28), datetime.date(2018, 6, 1), datetime.date(2018, 1, 1), 'Patient/e80dda2c-a260-dbf7-3167-bf0945f3a91d', 'Encounter/029d1814-d7bf-0624-524d-7ccda5f320f6', '029d1814-d7bf-0624-524d-7ccda5f320f6', 'female', 'black or african american', 'not hispanic or latino', '672')
```

Comment:
This changes were we pull the end of an encounter from, which is null in our test data set. I'll verify this out of thread, but maybe we should coalesce to start date if end date is null?